treeherder/intermittents_commenter/commenter.py

```diff
@@ -452,15 +450,12 @@ def build_bug_map(self, bugs, option_collection_map, start_day, end_day):
             if test_name:
                 manifest = all_tests[test_name][0]
                 if manifest:
-                    """
-                    # Bug 1972307 - disabled due to no comments
-                    tasks_count = self.get_task_labels_and_count(manifest, start_day, end_day)
+                    tasks_count = self.get_task_labels_and_count(manifest, days)
```

Comment:
could we move this call to `self.get_task_labels_and_count` outside of the for loop near the top of this function?  I think we are building the same data structure each time through the loop.

Reply:
The task labels are the same, but the counts depend on the manifest, no ?